code/datums/traits/negative.dm

```diff
@@ -424,6 +424,39 @@
 	to_chat(quirk_holder, "<span class='boldannounce'>Your [slot_string] has been replaced with a surplus prosthetic. It is fragile and will easily come apart under duress. Additionally, \
 	you need to use a welding tool and cables to repair it, instead of bruise packs and ointment.</span>")
 
+/datum/quirk/nugget
+	name = "Nugget"
+	desc = "An accident caused you to lose ALL of your limbs. There's no way your insurance payed for all those prosthetics!"
+	value = -1
+	var/slot_string = "limb"
+
+/datum/quirk/nugget/on_spawn()
+	var/mob/living/carbon/human/nuggeted = quirk_holder
+	for(var/X in nuggeted.bodyparts)
+		var/obj/item/bodypart/BP = X
+		if(BP.body_part != HEAD && BP.body_part != CHEST)
+			if(BP.dismemberable)
+				BP.dismember()
+	if(nuggeted.buckled)
+		nuggeted.buckled.unbuckle_mob(nuggeted)
+	nuggeted.suppress_bloodloss(1800) //stop them from bleeding out
```

Comment:
Is this a timed proc?
Or if it's not, does it have any affect on a living person that is then treated?

Reply:
It simply de-limbs you upon spawn in comedic fashion.  If you manage to get patched up, you can go on to have a normal round.  If that's worth the 1 point, they can have at it, haha.  Or if even that is a problem for some reason I can make it a 0 point quirk.